docs/resources/azurerm_ad_users.md.erb

```diff
@@ -10,6 +10,10 @@ some or all Azure Active Directory users within a Tenant.
 
 <br />
 
+## Azure REST API version
+
+This resource interacts with version `1.6` of the Azure Graph API. For more information see the [official Azure documentation](https://msdn.microsoft.com/Library/Azure/Ad/Graph/api/users-operations#GetUsers).
```

Comment:
I guess I expected this section to include something about how the documentation should be referenced with attention to specific resource API versions...

Reply:
As far as I can tell there is no way to select the version of the documentation you want. 😦